feat(nova): ✨ add role-based visibility for menu items OC: 6041

carlacupani opened the pull request:
- Implement role-based visibility for menu items in the Nova service provider.
- Restrict access to 'App', 'NovaUser', and 'Media' resources to users with the 'Administrator' role.
- Ensure external links 'Horizon' and 'Telescope' are also restricted to 'Administrator' role.
- Maintain existing email-based access control for the 'Tools' menu section.

carlacupani committed:
feat(nova): ✨ add role-based visibility for menu items OC: 6041

- Implement role-based visibility for menu items in the Nova service provider.
- Restrict access to 'App', 'NovaUser', and 'Media' resources to users with the 'Administrator' role.
- Ensure external links 'Horizon' and 'Telescope' are also restricted to 'Administrator' role.
- Maintain existing email-based access control for the 'Tools' menu section.

carlacupani committed:
fix(provider): 🐛 remove redundant comma in NovaServiceProvider OC:6041

- Eliminated an unnecessary comma after the 'canSee' closure in the MenuSection::make method.
- This change ensures the correct syntax and prevents potential syntax-related issues in the application.

peppedeka commented on app/Providers/NovaServiceProvider.php:
```diff
@@ -58,8 +61,10 @@ public function boot(): void
                 // ])->icon('document'),
 
                 MenuSection::make('Tools', [
```
penso vada nascosto tutta la sezione tools

carlacupani committed:
refactor(nova): ♻️ simplify menu item visibility condition OC:6041

- Removed the condition that restricts menu visibility to users with a specific email.
- Maintained the icon display for the briefcase menu item without the email check, ensuring that role-based visibility is still respected.
